code/game/machinery/doors/windowdoor.dm

```diff
@@ -188,6 +188,17 @@
 			qdel(src)
 			return
 
+	if(I.iscrowbar() && user.a_intent == I_HELP)
+		if(inoperable())
+			visible_message("\The [user] forces \the [src] [density ? "open" : "closed"].")
```

Comment:
This should probably have a sound.

Reply:
Thought I responded to this already.
They currently use the same sound as when they open & close normally. I did consider alternative sounds but the one I thought of, the same that airlocks use, didn't feel fitting for the windoors.